proto/osmosis/swaprouter/v1beta1/query.proto

```diff
@@ -4,6 +4,7 @@ package osmosis.swaprouter.v1beta1;
 import "gogoproto/gogo.proto";
 import "osmosis/swaprouter/v1beta1/genesis.proto";
 import "osmosis/swaprouter/v1beta1/tx.proto";
+import "osmosis/swaprouter/v1beta1/swap_route.proto";
```

Comment:
Why does swap route proto get added here to query?

Reply:
https://github.com/osmosis-labs/osmosis/pull/3525/files#r1033004282